apollo-runtime/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
   compile dep.supportAnnotations
   compile dep.okHttp
   compile dep.moshi
-  provided project(":apollo-api")
+  compile project(":apollo-api")
```

Comment:
I had to change this to compile for the test to work (`SQLStoreTest` now needs access to `Optional` in `:apollo-api` @marwanad any reason this needed to be `provided`?